lib/mongo/uri.rb

```diff
@@ -647,6 +646,37 @@ def zlib_compression_level(value)
       nil
     end
 
+    # Parses the max pool size.
+    #
+    # @param value [ String ] The max pool size string.
+    #
+    # @return [ Integer | nil ] The min pool size if it is valid, otherwise nil (and a warning will)
```

Comment:
The closing paren here is a bit too early in the sentence.

Reply:
Oops! Good catch